AirportTicketBookingSystem/AirportTicketBookingSystem.csproj

```diff
@@ -5,6 +5,20 @@
     <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+	 <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
   </PropertyGroup>
 
+  <ItemGroup>
+	 <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+	 <PackageReference Include="AutoFixture" Version="4.18.1" />
```

Comment:
these packages are not needed in the main project, just have them in the tests project csproj